src/runtime/HalideRuntime.h

```diff
@@ -1890,6 +1890,25 @@ HALIDE_ALWAYS_INLINE halide_type_t halide_type_of<int64_t>() {
     return halide_type_t(halide_type_int, 64);
 }
 
+// Some programming environments use aliases like `typedef long long int64`
+// (rather than the C99 integer types); for various reasons, these aren't
+// always considered the same type by C++, but we'd like to quietly accept them
+// in Halide, so add mappings here to the values we expect (along with static_asserts
+// to verify that the situation remains as we expect).
+template<>
+HALIDE_ALWAYS_INLINE halide_type_t halide_type_of<long long>() {
```

Comment:
Does this work out OK in environments in which int64_t == long long? I guess the buildbots will tell us.